draft-ietf-rats-corim.md

```diff
@@ -458,7 +470,23 @@ The following describes each child item of this group.
 
 # CoMID {#sec-comid}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/7
+A Concise Module Indentifier (CoMID) contains tag that convey information about hardware or firmware component or module used by supply chain entities to provision into a verifier. Each CoMID has a unique tag identity which is used to identify the CoMID unambigously with in the Verifier.
```

Comment:
"...is a tag that..."
The tag isn't a conveyance mechanism. Suggest: "...tag contains information about..."
"...used by supply chain <to the rest of the sentence>" can be removed since it is out of scope for this document.
Suggest: "Each CoMID tag has a unique identifier that is used to identify tag instances."